Detector/DetSensitive/DetSensitive/DigitalCalorimeterSD.h

```diff
@@ -0,0 +1,201 @@
+#ifndef DETSENSITIVE_DIGITALCALORIMETERSD_H
+#define DETSENSITIVE_DIGITALCALORIMETERSD_H
+
+// DD4hep
+#include "DDG4/Geant4Hits.h"
+#include "DDSegmentation/Segmentation.h"
+//#include "DD4hep/objects/SegmentationsInterna.h"
+
+// Geant
+#include "G4VSensitiveDetector.hh"
+#include "G4THitsCollection.hh"
+
+#include <fstream>
+#include <string>
+
+#include "TFile.h"
+#include "TH1F.h"
+
+/** DigitalCalorimeterSD DetectorDescription/DetSensitive/src/DigitalCalorimeterSD.h DigitalCalorimeterSD.h
+ *
+ *  Simple sensitive detector for calorimeter.
+ *  It is based on DD4hep::Simulation::Geant4GenericSD<Calorimeter> (but it is not identical).
+ *  In particular, the position of the hit is set to G4Step::GetPreStepPoint() position.
+ *  New hit is created for each energy deposit.
+ *  No timing information is saved.
+ *
+ *  @author    Anna Zaborowska
+ */
+
+struct Strixel
```

Comment:
According to [FCC coding style](https://github.com/HEP-FCC/FCCSW/blob/master/doc/CppCodingStyleGuidelines.md) we start variables with lower case letters.
And run `checkformat` to fix the formatting.